dungeon/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "dungeon",
+  "assets": [
+    {
+      "description": "Dungeon native token",
+      "denom_units": [
+        {
+          "denom": "udgn",
+          "exponent": 6
+        }
+      ],
+        {
```

Comment:
Please remove these lines:
```
   { 
      "denom": "udgn",
      "exponent": 0
    },
    {
      "denom": "DGN",
      "exponent": 6
    }
  ],
```

Reply:
This advise was counterproductive. If these lines are added back in it may be able to pass validation checks.